docs/harfbuzz-sections.txt

```diff
@@ -350,6 +350,7 @@ hb_coretext_font_set_funcs
 <SECTION>
 <FILE>hb-directwrite</FILE>
 hb_directwrite_face_create
+hb_directwrite_face_get_dw_font_face
```

Comment:
Should the old symbols be moved to the deprecated section?

Reply:
I noticed that we didn’t do this for the deprecated `hb_graphite2_font_get_gr_font()`, so I thought may be we do that only for core API. Should I move the three of them.

Reply:
I moved them in the documentation. Do we want me to move the declerations to `hb-deprecated.h` too?

Reply:
Since `hb-deprecated.h` is included by `hb.h`, I don't see why not.

Reply:
But `hb-ot.h`, `hb-ft.h`, etc. are not included by `hb.h`, which I think why `hb-ot-deprecated.h` exists (to be included from `hb-ot.h`.

Reply:
In that sense, I think including the deprecated `hb_ot_*` APIs in the `hb-deprecated` section in the documentation is not strictly accurate, ad that section’s includes has only `hb.h` which does not provide the `hb_ot_*` APIs.

Reply:
> But `hb-ot.h`, `hb-ft.h`, etc. are not included by `hb.h`, which I think why `hb-ot-deprecated.h` exists (to be included from `hb-ot.h`.

Oh right. We don't want to pull in graphite ft etc headers in hb-deprecated.h; leave them then. Thanks.